### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by jackra1n**
I took a look at the action runs after LMU run them and think those are some good small improvements :) 
- Update deprecated actions
- Set `workDir` variable in env
- show migrated repos in a separate step (easier to find)
- Extract function
- Ingore self for raising

### CODE REVIEW

File: raise-all-market-products.sh [hunk 1]
```diff
@@ -35,15 +36,19 @@ collectRepos() {
     .name'
 }
 
+showMigratedRepos() {
+  if [ -f "${workDir}/migrated-repos.txt" ]; then
```
Comment (ivy-rew) [inline raise-all-market-products.sh:40]: minor duplication: I'd assign the file to a variable and re-use it
log="${workDir}/migrated-repos.txt"

**Review by ivy-rew — APPROVED** [review]

File: .github/workflows/raise-products.yml [hunk 2]
```diff
@@ -18,13 +18,16 @@ on:
 jobs:
   raise-products:
     runs-on: ubuntu-latest
+    env: 
+      workDir: $(pwd)
```
Comment (ivy-rew) [inline .github/workflows/raise-products.yml:22]: :love_letter:  cool

File: .github/workflows/raise-products.yml [hunk 3]
```diff
@@ -39,7 +42,11 @@ jobs:
       - name: Raise Products
         run: |
           export engineUrl=${{ github.event.inputs.engineUrl }}
-          export workDir=$(pwd)
           export IVY_JAVA_HOME=$JAVA_HOME_17_X64
           ./raise-all-market-products.sh ${{ github.event.inputs.version }} ${{ github.event.inputs.product }}
         shell: bash
+
+      - name: Show migrated repositories
+        if: ${{ github.event.inputs.product }} == ''
```
[the remark this answers is not in the document]
Comment (jackra1n) [reply .github/workflows/raise-products.yml:50]: If its empty it raises all repos
So we only show migrated repos if you raise all

**Comment by ivy-rew** [comment]
thanks :1st_place_medal:

**Pull request merged**